### PR TITLE
Retrieve Format property when serializing as V2

### PR DESCRIPTION
**Pull request by irvinesunday**
Patch fix to @MaggieKimani1 branch

### CODE REVIEW

File: src/Microsoft.OpenApi/Models/OpenApiSchema.cs [hunk 1]
```diff
@@ -630,6 +630,10 @@ internal void WriteAsItemsProperties(IOpenApiWriter writer)
             }
 
             // format
+            Format ??= AllOf?.FirstOrDefault(x => x.Format != null)?.Format ??
+                    AnyOf?.FirstOrDefault(x => x.Format != null)?.Format ??
+                    OneOf?.FirstOrDefault(x => x.Format != null)?.Format;
```
Comment (baywet) [inline src/Microsoft.OpenApi/Models/OpenApiSchema.cs:635]:
```suggestion
            Format ??= AllOf?.FirstOrDefault(static x => x.Format != null)?.Format ??
                    AnyOf?.FirstOrDefault(static x => x.Format != null)?.Format ??
                    OneOf?.FirstOrDefault(static x => x.Format != null)?.Format;
```

**Review by baywet — CHANGES_REQUESTED** [review]

**Comment by baywet** [comment]
Can we get a preview release for this one soon so we can test the other fix that depends on it in the conversion library please?

**Comment by MaggieKimani1** [comment]
This was actually supposed to go out with v1.4.0 which I'm to release tomorrow.

**Comment by baywet** [comment]
We can also wait until tomorrow ;-)

**Pull request merged**